playbooks/infra/deploy-vm-bastion-libvirt.yml

```diff
@@ -186,6 +237,11 @@
             uuid: ""
             vcpu: "{{ vm_cpu }}"
 
+    - name: Start a bastion
```

Comment:
Do we need to start? create_vms role starts VM, no?

Reply:
so it's only defined it [here](https://github.com/eifrach/ansible-collection-redhatci-ocp/blob/b6d75ffd1efa71258da64a62bb2f2eed28fb1bac/roles/create_vms/templates/create_vm.sh.j2#L44)

we need to manually start it.

previously it was working cause the `print-xml` didn't ran so it created the VM